common/rakelib/utils.rake

```diff
@@ -17,7 +18,7 @@ class Utils
   # run a command using xcrun and xcpretty if applicable
   def self.run(cmd, task, subtask = '', xcrun: false, formatter: :raw)
     commands = xcrun ? [*cmd].map { |cmd|
-      "#{version_select} xcrun #{cmd}"
+      "#{@@version_select} xcrun #{cmd}"
```

Comment:
The concept of the memoization is to still always call the method — which will either return immediately with the cached version, or compute it and cache it if it's the first time it's requested.

So donc change the call sites, otherwise you risk using an empty value if the value was never cached